virttest/shared/deps/rss/setuprss.bat

```diff
@@ -1,5 +1,23 @@
+rem Enable delayed expansion to be able to use !ERRORLEVEL!
+setlocal enabledelayedexpansion
 set rsspath=%1
-if [%1]==[] set rsspath=%~dp0\rss.exe
+
+if [%1]==[] (
+    echo "Guessing rss path to use based on processor architecture"
+    :: Not using the PROCESSOR_ARCHITECTURE environment variable directly here as there are situations in Win7
+    :: install where the installation will be x64 but the environment variables, and the file system at
+    :: that point, are in a 32-bit state and %PROCESSOR_ARCHITECTURE% will return x86 on AMD64 hardware.
+    :: Since this file is called during the unattended installation, it is best to access the variable from
+    :: the registry instead. More information: https://github.com/avocado-framework/avocado-vt/pull/2920
```

Comment:
Perhaps it is better to point to the commit hash within git? Otherwise the GitHub link is quite platform specific.

Reply:
The idea is to point to the discussion that took place regarding the comment (the link was missing the comment part), providing extra information in external sources. There is no commit to point to for more info in this case

Reply:
Oh, I understand better now